rskj-core/src/test/java/org/ethereum/jsontestsuite/BlockTestingCase.java

```diff
@@ -26,7 +26,7 @@
 import java.util.List;
 import java.util.Map;
 
-public class BlockTestCase {
+public class BlockTestingCase {
```

Comment:
do we really need this renaming? It has generated lots of changes :). `Test case` still might be a better [wording](https://en.wikipedia.org/wiki/Test_case)

Reply:
hmm.. I see. What about marking it with `// NOSONAR` ?..

Reply:
.. anyway I'm okay with either option